locksmith/src/operations/wedlocksOperations.ts

```diff
@@ -220,11 +232,27 @@ const getTemplates = ({
     ? [`keyAirdropped${lockAddress.trim()}`, `keyAirdropped`]
     : [`keyMined${lockAddress.trim()}`, 'keyMined']
 }
+
+const isSendEmailEnabled = async (
+  lockAddress: string,
+  network?: number
+): Promise<boolean> => {
+  if (lockAddress && network) {
+    const settings = await lockSettingOperations.getSettings({
+      lockAddress,
+      network,
+    })
+    // fallback to true when settings are not present
+    return settings?.sendEmail ?? true
+  }
+  return true
+}
```

Comment:
as commented earlier. Maybe create a default settings object which is returned if nothing setting found for the lock and use it for getting sendEmail.